CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 2.25.0 (IN PROGRESS)
 
 * Implement exporting of only overdue loans for overdue loans report. Refs UIU-1121.
+* Create a footer, using <PaneFooter> component, and pass it as an optional prop for <Pane> component. Refs STCOM-429.
```

Comment:
```suggestion
* Implement footer with _Save & close_ and _Canclel_ buttons on the edit user form. Refs STCOM-429.
```

Please take out the implementation details and be more specific about the feature.